chroot/run.go

```diff
@@ -901,6 +901,13 @@ func setupChrootBindMounts(spec *specs.Spec, bundlePath string) (undoBinds func(
 		return err
 	}
 
+	// Create an empty directory for possible use later on.
+	roEmptyDir := filepath.Join(bundlePath, "empty")
+	if err := os.Mkdir(roEmptyDir, 0700); err != nil {
+		return undoBinds, errors.Wrapf(err, "error creating empty directory %q", roEmptyDir)
+	}
+	removes = append(removes, roEmptyDir)
```

Comment:
removes and roEmptyDir aren't used until the for statement at 990 and there are a few possible errors between here and there.  Does it make sense to move this chunk down just before that for statement?

Reply:
No harm in doing so.  Changing it.